ethereum/retesteth/src/main/java/org/hyperledger/besu/ethereum/retesteth/methods/TestMineBlocks.java

```diff
@@ -72,6 +72,7 @@ private boolean mineNewBlock() {
             Functions.identity(),
             context.getEthHashSolver(),
             Wei.ZERO,
+            0.0,
```

Comment:
Is the ratio at 0 normal in this case?

Reply:
Yeah all parameters are neutral for this test. Setting it to 0 avoids some unexpected behaviour here